docs/user/user/metrics/data-control-plane/product-integration.md

```diff
@@ -80,7 +80,7 @@ features:
 Once the Feature Variable has been defined, the final step is to fetch the configuration from Nimbus and supply it to the Glean API. This can be done during initialization and again any time afterwards, such as in response to receiving an updated configuration from Nimbus. Only the latest configuration provided will be applied and any previously configured metrics that are omitted from the new configuration will not be changed. An example call to set a configuration from the “homescreen” Nimbus Feature could look like this:
 
 ```Swift
-Glean.setMetricsEnabledConfig(FxNimbus.features.homescreen.value().metricsEnabled)
+Glean.setRemoteSettingsConfig(FxNimbus.features.homescreen.value().metricsEnabled)
```

Comment:
I can see where there may be some confusion. I (re)named this to reflect the _source_ of the the config that was to be supplied here, but it might be misconstrued that this is somehow setting some internal remote-settings setup?

Let me ponder on what might be a more concise name for this...

Reply:
I've settled on an entirely different verb and a internally focused noun to disambiguate this from RemoteSettings: `applyServerKnobsConfig`